shared/translation-manager.js

```
@@ -0,0 +1,210 @@
+/**
+ * 共有翻訳マネージャー
+ * 拡張機能全体で翻訳システムを一元管理するシングルトンクラス
+ */
+class TranslationManager {
+  constructor() {
+    if (TranslationManager.instance) {
+      return TranslationManager.instance;
+    }
+
+    this.currentTranslations = {};              // 現在の翻訳データ
+    this.currentLanguage = 'ja';                // 現在の言語設定
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)